src/dealWizard/components/proposalLeadSection/proposalLeadSection.ts

```diff
@@ -1,9 +1,38 @@
 import { bindable } from "aurelia-typed-observable-plugin";
 import { IProposalLead } from "entities/DealRegistrationTokenSwap";
 import { WizardErrors } from "services/WizardService";
+import "./proposalLeadSection.scss";
```

Comment:
Why have proposalLeadSection if only used once?

Reply:
ok, this can be removed once you have just a single stage component

Reply:
This component is actually used three times throughout the app. Or what do you mean?
![image](https://user-images.githubusercontent.com/4083652/152679455-19a273df-234b-4fd2-8974-fa27646f17eb.png)